MITRE ATT&CK for Azure Sentinel.ipynb

```diff
@@ -0,0 +1,2063 @@
+{
```

Comment:
<p>Doesn't work if ./Layers/ doesn't exist.</p><p>Consider an os.makedirs check to create the folder if needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>